Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Send/Settings/ToSpeckleSettingsManager.cs

```diff
@@ -86,55 +103,84 @@ out ReferencePointType referencePoint
       return currentTransform;
     }
 
-    throw new ArgumentException($"Invalid reference point value: {referencePointString}");
-  }
-
-  public bool GetSendParameterNullOrEmptyStringsSetting(SenderModelCard modelCard)
-  {
-    var value = modelCard.Settings?.FirstOrDefault(s => s.Id == "nullemptyparams")?.Value as bool?;
-    var returnValue = value != null && value.NotNull();
-    if (_sendNullParamsCache.TryGetValue(modelCard.ModelCardId.NotNull(), out bool? previousValue))
-    {
-      if (previousValue != returnValue)
-      {
-        EvictCacheForModelCard(modelCard);
-      }
-    }
+    // log the issue
+    _logger.LogWarning(
+      "Invalid reference point setting received: '{ReferencePointString}' for model {ModelCardId}, using default: {DefaultValue}",
+      referencePointString,
+      modelCard.ModelCardId,
+      SendReferencePointSetting.DEFAULT_VALUE
+    );
 
-    _sendNullParamsCache[modelCard.ModelCardId] = returnValue;
-    return returnValue;
+    // return default (null for InternalOrigin means no transform)
+    _referencePointCache[modelCard.ModelCardId.NotNull()] = null;
+    return null;
```

Comment:
Reference point is the only setting where we're returning `null` instead of the settings default value...
is that ok?